src/pages/Categories.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+  const handleFilter = (e) => {
+    dispatch(e.target.value);
+  };
 
-function Categories() {
   return (
-    <button type="button">Check Status</button>
+    <button type="button" className="btn btn-primary" onClick={handleFilter}>Check Status</button>
   );
-}
+};
 
 export default Categories;
```

Comment:
- [ ] Kindly use the `checkStatus` method that dispatch the check status action also you should get the categories state to display it.

```javascript
import React from 'react';
import { useDispatch, useSelector } from 'react-redux';
import { checkStatus } from '../redux/categories/categories';

const Categories = () => {
  const category = useSelector((state) => state.categories);
  const dispatch = useDispatch();
  const handleFilter = (e) => {
    dispatch(checkStatus(e.target.value));
  };

  return (
    <>
      <button type="button" className="btn btn-primary" onClick={handleFilter}>Check Status</button>
      <p>{category}</p>
    </>
  );
};

export default Categories;
```